src/build/__main__.py

```diff
@@ -78,8 +78,8 @@ def _error(msg: str, code: int = 1) -> NoReturn:  # pragma: no cover
     :param msg: Error message
     :param code: Error code
     """
-    print('{red}ERROR{reset} {}'.format(msg, **_STYLES))
-    exit(code)
+    print('{red}ERROR{reset} {}'.format(msg, **_STYLES), file=sys.stderr)
```

Comment:
```suggestion
    print('{red}ERROR{reset} {}'.format(msg, **_STYLES))
```

We can change it out for stderr later.